src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -453,8 +497,15 @@ module.exports = function<T, P, I, TI, C>(
       case HostComponent:
         return updateHostComponent(current, workInProgress);
       case HostText:
-        // Nothing to do here. This is terminal. We'll do the completion step
-        // immediately after.
+        const newText = workInProgress.pendingProps;
```

Comment:
Technically didn't have to move this one but did this for consistency.
Can move it back too.